packages/nimble-components/.storybook/manager.js

```diff
@@ -7,9 +7,10 @@ addons.setConfig({
         filters: {
```

Comment:
Wanna add a change in this PR to mark the ready key label-hidden in the component status table?

![image](https://github.com/ni/nimble/assets/1588923/21b7a9eb-c71c-4a00-9085-81b66697a694)